packages/eds-core-react/src/components/Popover/PopoverTitle.tsx

```diff
@@ -1,23 +1,11 @@
 import { forwardRef, HTMLAttributes } from 'react'
 import styled, { css } from 'styled-components'
-import { Divider } from '../Divider'
 
 const StyledPopoverTitle = styled.div(({ theme }) => {
   return css`
     margin-right: ${theme.entities.closeButton.width};
     max-width: calc(${theme.maxWidth} - ${theme.entities.closeButton.width});
```

Comment:
It seems like the margin-right here was to take into account the hardcoded close button and can now be removed (and some margin-left can be added to the manually added closebutton instead.
Also the max-width calc can be replaced with just the max-width for same reasons?

Reply:
Thanks @oddvernes.